client/my-sites/upgrades/domain-management/transfer/pending-transfer.jsx

```diff
@@ -0,0 +1,125 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+import Button from 'components/button';
+import { acceptTransfer, declineTransfer } from 'lib/upgrades/actions';
+import notices from 'notices';
+import { translate } from 'lib/mixins/i18n';
+
+const somethingWrongMessage = translate(
+	'Oops! Something went wrong and your request could not be ' +
+	'processed. Please try again or {{a}}Contact Support{{/a}} if ' +
+	'you continue to have trouble.',
+	{
+		components: {
+			a: (
+				<a
+					href="https://support.wordpress.com/contact/"
+					target="_blank" />
+			)
+		}
+	}
+);
+
+const Unlocked = React.createClass( {
```

Comment:
Copy-paste error? Instead of `Unlocked` it should be `PendingTransfer`. Not that it matters _that_ much, but I was confused for a second "Wait, I swear I've already reviewed an `Unlocked` component....").